packages/devtools_app/lib/src/shared/config_specific/import_export/_export_web.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// ignore_for_file: avoid-unused-ignores, package:web upgrade is in progress.
+
```

Comment:
@srujzs FYI the DCM check started failing on the `avoid-unnecessary-type-casts` in this file. I ignored the
`avoid-unused-ignores` check instead of trying to fix the type casts since you are actively working on upgrading our package:web dep

Reply:
I've landed the pin to 0.4.0, so we don't need this anymore!